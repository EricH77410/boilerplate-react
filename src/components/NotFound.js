import React from 'react';
import { Link } from 'react-router-dom'
const NotFound = () => (
    <div>
        <h1>404</h1>
        <Link to="/">Home</Link>
    </div>
    
)

export default NotFound