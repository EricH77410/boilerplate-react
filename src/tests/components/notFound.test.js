import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';

test('should render NotFound page', ()=>{
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot()
})