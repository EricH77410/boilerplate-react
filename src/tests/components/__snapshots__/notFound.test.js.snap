// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound page 1`] = `
<div>
  <h1>
    404
  </h1>
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
